src/users/dto/create-user.dto.ts

```diff
@@ -22,4 +24,13 @@ export class CreateUserDto {
   @MinLength(4, { message: 'Password is too short' })
   @MaxLength(72, { message: 'Password is too long' })
   password: string;
+
+  @IsString()
+  partner: string;
+
+  @IsString()
+  @IsDefined()
+  status: PartnerStatusEnum;
+
+  requestedAt: Date;
```

Comment:
i think we don't need the infos, `partner`, `status`, and `requestedAt,` when users signup. what do you think?

Reply:
I cannot update requestedAt in usersService since its not a known property. Would it be acceptable to move requestedAt to updateUserDto?